tests/unit/controllers/submit-test.js

```diff
@@ -603,52 +694,59 @@ skip('changeInitialState', function() {
     // });
 });
 
-test('lockNode', function(assert) {
+test('finishUpload', function(assert) {
     const ctrl = this.subject();
     assert.equal(ctrl.get('nodeLocked'), false);
-    ctrl.send('lockNode');
-    assert.equal(ctrl.get('nodeLocked'), true);
-});
 
-skip('finishUpload', function() {
-    // test('finishUpload', function(assert) {
-    //     //TODO finishUpload calls 'next' action. Haven't gotten 'next' working in tests yet
-    // })
-});
+    Ember.run(() => {
+        ctrl.send('finishUpload');
 
+        assert.equal(ctrl.get('nodeLocked'), true);
+        assert.equal(ctrl.get('file'), null);
 
-skip('existingNodeExistingFile', function() {
-    // test('existingNodeExistingFile', function(assert) {
-    //     // TODO Many actions get called by this action. Sending POST to localhost:7357/nodeTags
-    //     // Getting Assertion Failed: You can only unload a record which is not inFlight
-    //     this.inject.service('store');
-    //     let store = this.store;
-    //     const ctrl = this.subject();
-    //     Ember.run(() => {
-    //         let node = store.createRecord('node', {
-    //             title: 'hello',
-    //             tags: ['first tag'],
-    //             description: 'The best abstract'
-    //         });
-    //         ctrl.set('nodeTitle', 'New title')
-    //         ctrl.set('node', node);
-    //         ctrl.send('existingNodeExistingFile');
-    //         assert.equal(ctrl.get('node.title'), 'New title');
-    //         assert.equal(ctrl.get('basicsAbstract'), node.get('description'));
-    //     });
-    // });
+        Ember.run.cancelTimers();
+    });
 });
 
+// test('existingNodeExistingFile', function(assert) {
```

Comment:
skip instead of commented out test? When I put the first draft of tests in here, I was asked to use skip instead of commenting so they were more visible.